plugins/web-worker-offloading/helper.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Helpers for Web Worker Offloading.
+ *
+ * @since 0.1.0
+ * @package web-worker-offloading
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Gets configuration for Web Worker Offloading.
+ *
+ * @since 0.1.0
+ * @link https://partytown.builder.io/configuration
+ * @link https://github.com/BuilderIO/partytown/blob/b292a14047a0c12ca05ba97df1833935d42fdb66/src/lib/types.ts#L393-L548
+ *
+ * @return array<string, mixed> Configuration for Partytown.
+ */
+function wwo_get_configuration(): array {
+	$config = array(
+		'lib' => wp_parse_url( plugin_dir_url( __FILE__ ), PHP_URL_PATH ) . 'build/',
```

Comment:
There used to be `forward` in here, can you clarify why you removed it?

Reply:
Yes, because it was empty so it wasn't having any purpose.